fe/fe-core/src/main/java/com/starrocks/clone/DynamicPartitionScheduler.java

```diff
@@ -500,12 +500,13 @@ public void runOnceForTest() {
     protected void runAfterCatalogReady() {
         // Find all tables that need to be scheduled.
         long now = System.currentTimeMillis();
-        if ((now - lastFindingTime) > Math.max(300000, Config.dynamic_partition_check_interval_seconds)) {
+        long checkIntervalMs = Config.dynamic_partition_check_interval_seconds * 1000L;
+        if ((now - lastFindingTime) > Math.max(60000, checkIntervalMs)) {
```

Comment:
for better testing, reduce min schedule time from 5min to 1min but original config has not changed.